NetworkingProject/DetailModel/View/DetailsViewController.swift

```diff
@@ -9,14 +9,15 @@ final class DetailsViewController: UIViewController {
 
     var id = Int() // убрать в презентер
```

Comment:
```suggestion
    var id = Int() // убрать в презентер
```
убери раз комент такой стоит)